src/java.base/share/classes/java/lang/Runtime.java

```diff
@@ -48,7 +48,51 @@
  * <p>
  * An application cannot create its own instance of this class.
  *
+ * <h2><a id="shutdown">Shutdown Sequence</a></h2>
+ * <p>
+ * The Java Virtual Machine initiates the <i>shutdown sequence</i> in response
+ * to one of several events:
+ * <ol>
+ * <li>when the number of non-daemon threads drops to zero for the first time;</li>
+ * <li>when the {@link #exit Runtime.exit} or {@link System#exit System.exit} method is called; or</li>
+ * <li>when some external event occurs, such as an interrupt or a signal is received from
+ * the operating system.</li>
+ * </ol>
+ *
+ * <p>At the beginning of the shutdown sequence, the set of registered
+ * {@linkplain #addShutdownHook shutdown hooks}
+ * is established atomically. After this point, registration and de-registration
+ * of shutdown hooks with
+ * {@link #addShutdownHook addShutdownHook}
+ * and
+ * {@link #removeShutdownHook removeShutdownHook}
+ * is prohibited. The shutdown hooks are then started in some unspecified order,
+ * and they are allowed to run concurrently. Any already-existing threads and any
+ * newly started threads also continue to run concurrently during the shutdown sequence.
```

Comment:
Instead of "Any already-existing threads" then you could say "Existing {@linkplain Thread#isAlive() live} threads".